### PR TITLE
[UI] Notification Center revamped

### LINKED ISSUE #3781: [UI] Notification Center: Success notifications not displaying in correct filter
#### Current Behavior
See the recording

https://user-images.githubusercontent.com/7570704/128276081-13d8e504-98ae-4232-92f3-fb9fae605a52.mov

#### Desired Behavior
Success notifications should display under "Success" filter.

#### Environment

- Browser: Safari
- Host OS: Mac 
- Meshery Server Version: stable-v0.5.42
- Platform: Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by afzal442**
**Description**
Added 'ALL' notification to handlers and refactored the component

This PR fixes #3781

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thanks for this PR, @afzal442. Rename the PR title and remove the pr/draft label when ready for review, @afzal442.

File: ui/components/MesheryEventViewer.js [hunk 1]
```diff
@@ -11,37 +11,36 @@ const styles = (theme) => ({
...
-    );
-  }
+
+  const {
+    classes, eventVariant, eventSummary, eventDetails,
+  } = props;
+
+  return (
+    <NoSsr>
+      <>
```
Comment (Jashpatel1) [inline ui/components/MesheryEventViewer.js:32]: @afzal442 Is this empty tag required?

**Comment by Jashpatel1** [comment]
@afzal442 Can you add a video or screenshot of your changes?

File: ui/components/MesheryEventViewer.js (hunk 1 above)
Comment (afzal442) [reply ui/components/MesheryEventViewer.js:32]: yes, this is a kind of syntactic sugar. have a look here https://reactjs.org/docs/fragments.html#short-syntax. thanks.

**Review by warunicorn19 — COMMENTED** [review]
Hey @afzal442 if you could maybe put up a screen recording of the changes you have done.
The Notification center seems to remain unchanged

https://user-images.githubusercontent.com/53532328/130694424-0205590a-6624-4cb8-a9e7-b0547e05732c.mov

**Comment by warunicorn19** [comment]
And I think this might be because most of your changes were reverted when @sudo-NithishKarthik merged the master into this branch

**Comment by afzal442** [comment]
Nope, that didn't update anything in the result of this PR. we could have some discussion on it later today.

**Comment by leecalcote** [comment]
Thank you, @warunicorn19. You're feedback is spot on.

**Comment by afzal442** [comment]
@sudo-NithishKarthik @warunicorn19 any update on it plz?👽😊

**Comment by warunicorn19** [comment]
@afzal442, @sudo-NithishKarthik and I have been procrastinating to fix this for some time now😂, @sudo-NithishKarthik we really need fix this😅.

**Comment by sudo-NithishKarthik** [comment]
Closing this PR as it does not fix the issue. The issue is not straightforward, I am intending to fix it while bringing in Redux.

**Pull request closed without merging**